generators/typescript-v2/ast/src/typescript.ts

```diff
@@ -12,6 +12,27 @@ export function reference(arg: AST.Reference.Args): AST.Reference {
     return new AST.Reference(arg);
 }
 
+export function func(args: AST.Func.Args): AST.Func {
+    return new AST.Func(args);
+}
+
+export function method(args: AST.Method.Args): AST.Method {
+    return new AST.Method(args);
+}
+
+export function invokeFunction(args: AST.FunctionInvocation.Args): AST.FunctionInvocation {
+    return new AST.FunctionInvocation(args);
+}
+
+export function invokeMethod(args: AST.MethodInvocation.Args): AST.MethodInvocation {
+    return new AST.MethodInvocation(args);
+}
```

Comment:
Based on the other comments, I _think_ we only need these to start.

Reply:
up to you, I'm happy to toss them in now if you're up for reviewing them